style/style.css

```diff
@@ -1,5 +1,87 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&display=swap');
+
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
+  text-decoration: none;
+}
+
+body {
+  font-family: 'Inter', sans-serif;
+  color: #172b4d;
+}
+
+a {
+  color: inherit;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px;
+  margin-top: 10px;
+}
+
+.dark-blue {
+  color: #172b4d;
+}
+
+.sea-green {
+  color: #36b37e;
+}
+
+#logo {
+  font-weight: 600;
+  font-size: 20px;
+}
+
+#menu {
+  font-size: 20px;
+}
+
+#headline {
+  display: flex;
+  flex-direction: column;
+  background-image: url(/resource/header-illsutration-mobile.svg);
```

Comment:
- [ ] Please add `.` before your background image path to make your image work online because now it only works locally.
it suppose to be like that:
`background-image: url(./resource/header-illsutration-mobile.svg);`

Reply:
@Omar-Muhamad I'll be glad to change it but please let me hear from you if you have any other suggestions to achieve that same look

Reply:
@Omar-Muhamad Thank you very much. Your review is much appreciated.